src/components/DisplayParameters.astro

```diff
@@ -13,7 +13,7 @@ const { formats, direction } = Astro.props;
 
 <Tabs>
     {
-      (Object.entries(formats) as [keyof typeof formats, Format][]).map(([key, format]) => {
+      (Object.entries(formats) as [keyof typeof formats, Format][]).map(([_key, format]) => {
         
         // Specify the styling of each parameter
         return ( 
```

Comment:
More of a curiosity: In the next line React would like to have a `key` on `TabItem` like `<TabItem key={...}` to identify the elements of the resulting array. Omitting it would show a warning in the browsers dev tools at least in the development environment (not sure about production). Astro does not want that?

Reply:
I know, that's also why I couldn't mark the exact line :D
I was just curious, this is quite out of scope of this PR and likely needs to be checked in other places too (if it is even an issue at all).

Reply:
Not only does it now want that, it also doesn't allow you to set that, see https://starlight.astro.build/components/tabs/#tabitem-props. But then again, we're just generating static HTML, so I guess there's also no need to actually identify the elements.